packages/widgetbook/lib/src/fields/field_codec.dart

```diff
@@ -25,7 +25,8 @@ class FieldCodec<T> {
   /// print(encoded); // {foo:bar,baz:qux}
   /// ```
   static String encodeQueryGroup(Map<String, String> group) {
-    final pairs = group.entries.map((entry) => '${entry.key}:${entry.value}');
+    final pairs = group.entries
+        .map((entry) => '${Uri.encodeComponent(entry.key)}:${entry.value}');
```

Comment:
**todo:** For more readability, please extract the `Uri.encodeComponent(entry.key)` to a local variable named `encodedKey`. Same comment for decoding, let's use a `decodedKey` there.